homeassistant/components/ezviz/__init__.py

```diff
@@ -38,17 +37,16 @@
 ]
 
 
-async def async_setup_entry(hass, entry):
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Ezviz from a config entry."""
     hass.data.setdefault(DOMAIN, {})
 
     if not entry.options:
         options = {
-            CONF_FFMPEG_ARGUMENTS: entry.data.get(
-                CONF_FFMPEG_ARGUMENTS, DEFAULT_FFMPEG_ARGUMENTS
-            ),
-            CONF_TIMEOUT: entry.data.get(CONF_TIMEOUT, DEFAULT_TIMEOUT),
+            CONF_FFMPEG_ARGUMENTS: DEFAULT_FFMPEG_ARGUMENTS,
+            CONF_TIMEOUT: DEFAULT_TIMEOUT,
```

Comment:
Where would CONF_FFMPEG_ARGUMENTS and CONF_TIMEOUT be set in `data` instead of in `options`?

Reply:
Hi @emontnemery,

It's not set in data at all and that's the reason why I'm cleaning this up to just set options if it's missing.